quickstart/www/analytics/index.html

```diff
@@ -0,0 +1,2 @@
+<!-- Redirect to ../index.html -->
```

Comment:
Is this needed at all?

Reply:
Probably not. We'll look at removing it.

Reply:
Removed.